src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -277,7 +278,7 @@ public ObservableCollection<string> PackagePathsForInstall
         {
             get
             {
-                if (packagePathsForInstall == null || !packagePathsForInstall.Any())
```

Comment:
why make this change?

Reply:
It was needed for a while but not any more. I will revert.